README.md

```diff
@@ -244,6 +244,43 @@ decoded = encoding.msgpack_decode(base64.b64encode(note_field_bytes))
 print(decoded.dictify())
 ```
 
+### working with transaction group
+```python
+import params
+from algosdk import algod, kmd, transaction
+
+private_key_sender, sender = account.generate_account()
+private_key_receiver, receiver = account.generate_account()
+
+# create an algod and kmd client
+acl = algod.AlgodClient(params.algod_token, params.algod_address)
+kcl = kmd.KMDClient(params.kmd_token, params.kmd_address)
+
+# get suggested parameters
+params = acl.suggested_params()
+gen = params["genesisID"]
+gh = params["genesishashb64"]
+last_round = params["lastRound"]
+fee = params["fee"]
+
+# create a transaction
+amount = 10000
+txn1 = transaction.PaymentTxn(sender, fee, last_round, last_round+100, gh, receiver, amount)
+txn2 = transaction.PaymentTxn(receiver, fee, last_round, last_round+100, gh, sender, amount)
+
+# get group id and assign it to transactions
+gid = transaction.calculate_group_id([txn1, txn2])
```

Comment:
This might be cumbersome with a large number of txns. Please consider changing the function `calculate_group_id` to `assign_group_id([txn1, txn2,...])` that returns the list of txns with the tags already assigned.

Reply:
I also thought about it and it does not look practical - main use case is to group transactions signed by different keys, possible received from different people, so looping and signing would not be the main use case. My feeling calc function would be used more often than assign, so let's have both of them.

Reply:
Actually, here is a better idea than my previous one - let it take an optional Algorand address and output just the relevant ones. 

Also, we can definitely have both.

Reply:
done